src/govuk/settings/_colours-organisations.scss

```diff
@@ -15,132 +15,211 @@
+  "foreign-commonwealth-office":
+  (
+    colour: #003e74
   ),
-  "government-equalities-office": (
-    colour:  #9325b2
+  "foreign-commonwealth-development-office":
+  (
+    colour: #012169
   ),
-  "hm-government": (
-    colour: #0076c0,
-    colour-websafe: #347da4
+  "government-equalities-office":
+  (
+    colour: #005abb
   ),
-  "hm-revenue-customs": (
+  "hm-government":
+  (
+    colour: #266ebc
+  ),
+  "hm-revenue-customs":
+  (
     colour: #009390,
-    colour-websafe: #008670
+    colour-websafe: #007e7b
+  ),
+  "hm-treasury":
+  (
+    colour: #af292e
+  ),
+  "home-office":
+  (
+    colour: #9325b2
+  ),
+  "ministry-of-defence":
+  (
+    colour: #4d2942
   ),
-  "hm-treasury": (
-    colour: #af292e,
-    colour-websafe: #832322
+  "ministry-of-housing-communities-local-government":
+  (
+    colour: #009999,
+    colour-websafe: #008383
+  ),
+  "ministry-of-justice":
+  (
+    colour: #231f20
   ),
-  "home-office": (
-    colour: #9325b2,
-    colour-websafe: #9440b2
+  "northern-ireland-office":
+  (
+    colour: #002663
   ),
-  "ministry-of-defence": (
-    colour: #4d2942,
-    colour-websafe: #5a5c92
+  "office-of-the-advocate-general-for-scotland":
+  (
+    colour: #002663
   ),
-  "ministry-of-justice": (
-    colour: #231f20,
-    colour-websafe: #5a5c92
+  "office-of-the-leader-of-the-house-of-commons":
+  (
+    colour: #317023
   ),
-  "northern-ireland-office": (
-    colour: #002663,
-    colour-websafe: #3e598c
+  "office-of-the-leader-of-the-house-of-lords":
+  (
+    colour: #9c132e
   ),
-  "office-of-the-advocate-general-for-scotland": (
-    colour: #002663,
-    colour-websafe: #005ea5
+  "office-of-the-secretary-of-state-for-scotland":
+  (
+    colour: #002663
   ),
-  "office-of-the-leader-of-the-house-of-commons": (
-    colour: #317023,
-    colour-websafe: #005f8f
+  "office-of-the-secretary-of-state-for-wales":
+  (
+    colour: #a33038
   ),
-  "office-of-the-leader-of-the-house-of-lords": (
-    colour: #9c132e,
-    colour-websafe: #c2395d
+  "prime-ministers-office-10-downing-street":
+  (
+    colour: #0b0c0c
   ),
-  "scotland-office": (
-    colour: #002663,
-    colour-websafe: #405c8a
+  "scotland-office":
+  (
+    colour: #002663
   ),
-  "uk-export-finance": (
-    colour: #005747,
-    colour-websafe: #005ea5
+  "uk-export-finance":
+  (
+    colour: #d0132f
   ),
-  "uk-trade-investment": (
-    colour: #c80651,
-    colour-websafe: #005ea5
+  "uk-trade-investment":
+  (
+    colour: #c80651
   ),
-  "wales-office": (
-    colour: #a33038,
-    colour-websafe: #7a242a
+  "wales-office":
+  (
+    colour: #a33038
   )
 ) !default;
+
+/// Deprecated organisation colours
+///
+/// @type Map
+///
+/// A list of keys and deprecation messages, for organisations that have been
+/// dissolved and will be removed in a future version. Provide new keys for orgs
+/// that have rebranded.
+///
+/// @access private
+
+$_govuk-colours-organisations-deprecated: (
```

Comment:
Could I suggest instead adding this information to the existing objects in `$govuk-colours-organisations`?

So for example:

```scss
$govuk-colours-organisations: (
  // ...
  "department-for-business-energy-industrial-strategy": (
    colour: #003479,
    deprecated: true,
    message: "Dissolved in 2023, replaced by `department-for-business-trade`, `department-for-energy-security-net-zero` and `department-science-innovation-technology`."
  ),
  // ...
)
```

The `govuk-organisation-colour` function could then just check if `$org-colour` has the `deprecated` key.

Reply:
Is there any particular reason for two properties here vs. a single `deprecation-message` (or similar) property? 

Do you envisage a situation where we might output a message but not have one be deprecated?

Reply:
No, happy for them to be combined if you think that makes more sense 👍🏻